### PR TITLE
🌐 [i18n-ZH] Translate chat_templating.md into Chinese

### PR DESCRIPTION
**Pull request by shibing624**
# What does this PR do?

Translate chat_templating.md into Chinese
part of https://github.com/huggingface/transformers/issues/20095

## Who can review?
Documentation: @stevhliu 
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/zh/chat_templating.md [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+```
+
+可以看到，这一次tokenizer已经添加了[INST]和[/INST]来表示用户消息的开始和结束。
+Mistral-instruct是有使用这些token进行训练的，但BlenderBot没有。
+
+## 我如何使用聊天模板？
+
+正如您在上面的示例中所看到的，聊天模板非常容易使用。只需构建一系列带有`role`和`content`键的消息，
+然后将其传递给[`~PreTrainedTokenizer.apply_chat_template`]方法。
+另外，在将聊天模板用作模型预测的输入时，还建议使用`add_generation_prompt=True`来添加[generation prompt](#什么是"generation prompts"?)。
```
Comment (stevhliu) [inline docs/source/zh/chat_templating.md:69]:
```suggestion
另外，在将聊天模板用作模型预测的输入时，还建议使用`add_generation_prompt=True`来添加[generation prompt](#什么是generation-prompts)。
```

File: docs/source/zh/chat_templating.md [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+默认模板看起来像这样：
+
+```
+{% for message in messages %}
+    {{'<|im_start|>' + message['role'] + '\n' + message['content'] + '<|im_end|>' + '\n'}}
+{% endfor %}
+```
+
+
+如果您喜欢这个模板，下面是一行代码的模板形式，它可以直接复制到您的代码中。这一行代码还包括了[generation prompts](#什么是"generation prompts"?)，
```
Comment (stevhliu) [inline docs/source/zh/chat_templating.md:351]:
```suggestion
如果您喜欢这个模板，下面是一行代码的模板形式，它可以直接复制到您的代码中。这一行代码还包括了[generation prompts](#什么是generation-prompts)，
```

**Review by stevhliu — COMMENTED** [review]
Thanks for the translation! Aside from the broken links, it LGTM 🙂 

I can't review the content itself because I'm not a native speaker. Do you know anyone else in the community who would be interested in contributing and reviewing your work? Pinging @yyLeaves @jiaqiw09 who have translated Chinese docs in the past if you're interested!

**Comment by shibing624** [comment]
@yyLeaves @jiaqiw09  @bojone @yuanzhoulvpi please help to reviewing the translate work.

File: docs/source/zh/_toctree.yml [hunk 3]
```diff
@@ -37,6 +37,8 @@
     title: 使用特定于模型的 API
   - local: custom_models
     title: 共享自定义模型
+  - local: chat_templating
+    title: 聊天模板
```
Comment (windsonsea) [inline docs/source/zh/_toctree.yml:41]:
```suggestion
    title: 聊天模型的模板
```

File: docs/source/zh/chat_templating.md [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 聊天模板
```
Comment (windsonsea) [inline docs/source/zh/chat_templating.md:17]:
```suggestion
# 聊天模型的模板
```

File: docs/source/zh/chat_templating.md [hunk 5]
```diff
@@ -0,0 +1,437 @@
...
+
+## 介绍
+
+LLMs的一个越来越常见的用例是聊天。在聊天上下文中，模型不是继续单个文本字符串（就像标准语言模型一样），
```
Comment (windsonsea) [inline docs/source/zh/chat_templating.md:21]:
```suggestion
LLM 的一个常见应用场景是聊天。在聊天上下文中，不再是连续的文本字符串构成的语句（不同于标准的语言模型），
```

File: docs/source/zh/chat_templating.md [hunk 6]
```diff
@@ -0,0 +1,437 @@
...
+而是继续由一个或多个消息组成的对话，每个消息都包括一个角色，比如“用户”或“助手”，以及消息文本。
```
Comment (windsonsea) [inline docs/source/zh/chat_templating.md:22]:
```suggestion
聊天模型由一条或多条消息组成的对话组成，每条消息都有一个“用户”或“助手”等 **角色**，还包括消息文本。
```

File: docs/source/zh/chat_templating.md [hunk 7]
```diff
@@ -0,0 +1,437 @@
...
+```
+
+```text
+Conversation id: 76d886a0-74bd-454e-9804-0467041a63dc
+system: You are a friendly chatbot who always responds in the style of a pirate
+user: How many helicopters can a human eat in one sitting?
+assistant: Matey, I'm afraid I must inform ye that humans cannot eat helicopters. Helicopters are not food, they are flying machines. Food is meant to be eaten, like a hearty plate o' grog, a savory bowl o' stew, or a delicious loaf o' bread. But helicopters, they be for transportin' and movin' around, not for eatin'. So, I'd say none, me hearties. None at all.
+```
+
+[`ConversationalPipeline`]将负责处理所有的`tokenized`和调用`apply_chat_template`，一旦模型有了聊天模板，你只需要初始化pipeline并传递消息列表！
```
Comment (windsonsea) [inline docs/source/zh/chat_templating.md:143]:
```suggestion
[`ConversationalPipeline`]将负责处理所有的`tokenized`并调用`apply_chat_template`，一旦模型有了聊天模板，您只需要初始化pipeline并传递消息列表！
```
`你`和`您`可以统一一下

File: docs/source/zh/chat_templating.md [hunk 8]
```diff
@@ -0,0 +1,437 @@
...
+
+```
+{% if loop.last and add_generation_prompt %}
+{{ bos_token + 'Assistant:\n' }}
+{% endif %}
+```
+
+### 空格的注意事项
+
+尽可能地，我们已经尝试让Jinja忽略`{{ expressions }}`之外的空格。
```
Comment (windsonsea) [inline docs/source/zh/chat_templating.md:435]:
```suggestion
我们已经尽可能尝试让Jinja忽略除`{{ expressions }}`之外的空格。
```

File: docs/source/zh/chat_templating.md [hunk 9]
```diff
@@ -0,0 +1,437 @@
...
+然而，请注意Jinja是一个通用的模板引擎，它可能会将同一行上块之间的空格视为重要，并将其打印到输出中。
```
Comment (windsonsea) [inline docs/source/zh/chat_templating.md:436]:
```suggestion
然而，请注意Jinja是一个通用的模板引擎，它可能会将同一行文本块之间的空格视为重要，并将其打印到输出中。
```

**Review by windsonsea — COMMENTED** [review]
How about adding a space between English and Chinese, which would enhance the overall appearance.

**Review by shibing624 — COMMENTED** [review]
thanks，updated

File: docs/source/zh/chat_templating.md (hunk 7 above)
Comment (windsonsea) [reply docs/source/zh/chat_templating.md]: This shall be replaced from top to bottom rather than only this line

**Review by stevhliu — APPROVED** [review]
Thanks everyone!

**Pull request merged**